test/ruleLoaderTests.ts

```diff
@@ -18,6 +18,7 @@ import * as Lint from "./lint";
 describe("Rule Loader", () => {
     const path = require("path");
     const rulesDirectory = path.join(global.process.cwd(), "build/rules");
+    const rulesDirectories = [rulesDirectory];
```

Comment:
Done in commit https://github.com/palantir/tslint/commit/881616e80791bfbc0197ea5737fadbed04e257b5.